tests/console/krb5.pm

```diff
@@ -68,6 +70,9 @@ sub run {
     script_run "mkdir -p /run/user/`id -u tester`/krb5cc";
     script_run "chown tester:users /run/user/`id -u tester`/krb5cc";
 
+    # avoid failures in virtio-console due to unexpected PS1
+    assert_script_run('echo "PS1=\'# \'" >> ~tester/.bashrc') if check_var('VIRTIO_CONSOLE', '1');
```

Comment:
I thought the prompt should not even be overwritten since 33ce8e4ec . Maybe `SET_CUSTOM_PROMPT' is set in a scenario when it shouldn't? Or is this actually to workaround the issue of a default openSUSE/SLE prompt?

Reply:
set_standard_prompt is used with become_root or select_console when console is_activated, this test is doing just `su - tester`